CHANGELOG.md

```diff
@@ -51,7 +51,7 @@
 * [#3485](https://github.com/bbatsov/rubocop/issues/3485): Make OneLineConditional cop not register offense for empty else. ([@tejasbubane][])
 * [#3508](https://github.com/bbatsov/rubocop/pull/3508): Fix false negatives in `Rails/NotNullColumn`. ([@pocke][])
 * [#3462](https://github.com/bbatsov/rubocop/issues/3462): Don't create MultilineMethodCallBraceLayout offenses for single-line method calls when receiver spans multiple lines. ([@maxjacobson][])
-
+[#3514](https://github.com/bbatsov/rubocop/issues/3514) Changed the regular expression which defines normal case. ([@b-t-g])
```

Comment:
I noticed two issues currently filed that seem to be fixed by my changes. Should I refer to both even though the changelog messages for each issue would be roughly the same since the fix is the same (though I will elaborate on why the fix was made). 

For example, would this be an improvement?

"[#3514](https://github.com/bbatsov/rubocop/issues/3514) Make Style/VariableNumber cop not register an offense when variable names have an integer after the first "_"  but is a valid name when normal case is enabled ([@b-t-g][])"

Thanks for taking a look at this!